### PR TITLE
Add Arabic translation

### PR DESCRIPTION
**Pull request by daniserejo**
Add Arabic translation

### CODE REVIEW

**Review by camilavcoutinho — APPROVED** [review]

**Review by wdsrocha — APPROVED** [review]

**Review by wdsrocha — CHANGES_REQUESTED** [review]
Could you rebase it please?

**Comment by sheilagomes** [comment]
> Could you rebase it please?

Done!

**Review by camilavcoutinho — CHANGES_REQUESTED** [review]
I think something happened with the CHANGELOG

File: CHANGELOG.md [hunk 1]
```diff
@@ -33,8 +37,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - I18n Ca, Cs, Da, De, El, Fi, Fr, Ko, Nl, Pl, Ro, Ru, Sk, Sl, Sv and Uk.
 
-## [3.4.2] - 2021-08-27
-
```
Comment (wdsrocha) [inline CHANGELOG.md:37]: @sheilagomes I guess you moved this by mistake (now it's under the Arabic translation changelog). Could you move it back?

Comment (sheilagomes) [reply CHANGELOG.md]: Done! Sorry about that.

**Review by camilavcoutinho — APPROVED** [review]

**Review by wdsrocha — APPROVED** [review]
Thanks!!

**Pull request merged**